--- a/admin.local/lib/cassandra/conf/cassandra-env.sh	2017-09-18 10:21:55.644696692 +0200
+++ b/admin.local/lib/cassandra/conf/cassandra-env.sh	2017-09-18 10:22:49.530511725 +0200
@@ -161,8 +161,8 @@
 # times. If in doubt, and if you do not particularly want to tweak, go with
 # 100 MB per physical CPU core.
 
-MAX_HEAP_SIZE="4G"
-HEAP_NEWSIZE="8192M"
+MAX_HEAP_SIZE="1G"
+HEAP_NEWSIZE="1G"
 
 # Set this to control the amount of arenas per-thread in glibc
 #export MALLOC_ARENA_MAX=4
