--- ./admin/core/extensions/portal/api/plugins/CSWidget.php	2017-06-21 13:25:25.000000000 +0200
+++ ./admin/core/extensions/portal/api/plugins/CSWidget.php	2017-07-04 08:46:38.815008125 +0200
@@ -276,7 +276,7 @@
     $browserSupport = $metaData->getBrowserSupport();
 
     // Supported Browser
-    $browserSupport->addFirefoxSupport(52, 52, false, array('mac', 'win'), 'ESR');
+    $browserSupport->addFirefoxSupport(52, 54, false, $clients, 'ESR');
     $browserSupport->addMSIESupport(11, 11, false, array('win'));
     $browserSupport->addMSEdgeSupport(1, '', false, array('win'));
 
